### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
- Add PHP 7.3, 7.4 and 8.0 to choose from at install. #69
- Fix broken link

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: scripts/restore [hunk 1]
```diff
@@ -37,13 +37,18 @@ phpversion=$(ynh_app_setting_get --app=$app --key=phpversion)
...
 
 #=================================================
 # STANDARD RESTORATION STEPS
+#=================================================
+# REINSTALL DEPENDENCIES
+#=================================================
+ynh_script_progression --message="Reinstalling dependencies..." --weight=2
+
+ynh_install_app_dependencies "php${phpversion}-fpm"
+
```
Comment (yalh76) [inline scripts/restore:51]: Shouldn't we have something like:
```
if [ $phpversion != "none" ]
then
   blablabla
fi
```

File: scripts/restore [hunk 2]
```diff
@@ -102,7 +107,7 @@ chmod o-rwx "$final_path"
 # RESTORE THE PHP-FPM CONFIGURATION
 #=================================================
 
-ynh_restore_file --origin_path="/etc/php/$phpversion/fpm/pool.d/$app.conf"
+ynh_restore_file --origin_path="/etc/php/${phpversion}/fpm/pool.d/$app.conf"
```
Comment (yalh76) [inline scripts/restore:110]: Shouldn't we have something like:
```
if [ $phpversion != "none" ]
then
   blablabla
fi

File: scripts/restore [hunk 3]
```diff
@@ -115,7 +120,7 @@ ynh_restore_file --origin_path="/etc/php/$phpversion/fpm/pool.d/$app.conf"
 #=================================================
 ynh_script_progression --message="Reloading NGINX web server and PHP-FPM..."
 
-ynh_systemd_action --service_name=php$phpversion-fpm --action=reload
+ynh_systemd_action --service_name=php${phpversion}-fpm --action=reload
```
Comment (yalh76) [inline scripts/restore:123]: Shouldn't we have something like:
```
if [ $phpversion != "none" ]
then
   blablabla
fi

**Review by yalh76 — CHANGES_REQUESTED** [review]

**Review by yalh76 — DISMISSED** [review]

**Comment by yalh76** [comment]
!testme

**Review by yalh76 — APPROVED** [review]

**Comment by yalh76** [comment]
!testme

**Comment by tituspijean** [comment]
Let's gooooo!

File: scripts/install [hunk 4]
```diff
@@ -52,11 +53,23 @@ ynh_app_setting_set --app=$app --key=path --value=$path_url
...
 # STANDARD MODIFICATIONS
+#=================================================
+# INSTALL DEPENDENCIES
+#=================================================
+
+if [ $phpversion != "none" ]
+then
+    ynh_script_progression --message="Installing dependencies..." --weight=2
+
+    ynh_install_app_dependencies "php${phpversion}-fpm"
```
Comment (kay0u) [inline scripts/install:70]: IIRC `ynh_add_fpm_config` will install `php${phpversion}-fpm` if needed

File: scripts/backup [hunk 5]
```diff
@@ -51,7 +51,7 @@ ynh_backup --src_path="/etc/nginx/conf.d/$domain.d/$app.conf"
 # BACKUP THE PHP-FPM CONFIGURATION
 #=================================================
 
-ynh_backup --src_path="/etc/php/$phpversion/fpm/pool.d/$app.conf"
+ynh_backup --src_path="/etc/php/${phpversion}/fpm/pool.d/$app.conf"
```
Comment (kay0u) [inline scripts/backup:54]: This will probably crash if phpversion = "non"

**Review by kay0u — COMMENTED** [review]
I'm sorry I'm a bit late here :X

**Pull request merged**